wrapper/wrapperText.tsx

```diff
@@ -0,0 +1,13 @@
+import "./wrapper.css";
+
+function WrapperText(props: string) {
+  const string: string = props;
+
+  return (
+    <div className="blockWrap">
+      <p className="wrapText">{string}</p>
```

Comment:
а чо сразу из пропсов не подтянуть? 
```suggestion
      <p className="wrapText">{props.string}</p>
```